### ISSUE TITLE
cron job on toaster needs to listen to 9oose and respond, it's not running locally (it is in deploy.sh but only as a side effect)

### ISSUE THREAD

**Issue by yrmo** [yrmo/blog#6]

**Comment by yrmo** [comment]
Or fix #7

**Comment by yrmo** [comment]
`ping` doesnt work, bit hacky but you could use `curl` and do `wc -l` but pass `--silent` so just a number is output. With 9oose.com running:

```
(main) Ryans-MacBook-Air:blog ryan$ curl https://9oose.com curl --silent https://9oose.com | wc -l
   15430
```

When not running Nginx returns:

```
(main) Ryans-MacBook-Air:blog ryan$ curl https://9oose.com 
<html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.18.0 (Ubuntu)</center>
</body>
</html>
(main) Ryans-MacBook-Air:blog ryan$ curl https://9oose.com curl --silent https://9oose.com | wc -l
      14
```
